src/snowflake/cli/api/project/schemas/native_app/application.py

```diff
@@ -40,3 +46,7 @@ class Application(UpdatableModel):
         title="Whether to enable debug mode when using a named stage to create an application object",
         default=True,
     )
+    post_deploy: Optional[List[ApplicationPostDeployHook]] = Field(
+        title="Steps that will be executed after the application object is created/upgraded",
+        default=None,
+    )
```

Comment:
can we make this an empty list instead of None?

Reply:
It was my intuition at first too, but it's not consistent with other similar existing properties. This way we will also lose the notion of the user not specifying any value and specifying an empty list.
Any strong arguments for not keeping it `None`?

Reply:
nah, I was just trying to avoid None as much as possible, but differentiating between no list provided vs an explicit empty list makes sense